camel-idea-plugin/src/main/java/com/github/cameltooling/idea/runner/debugger/CamelDebuggerSession.java

```diff
@@ -802,7 +802,8 @@ private String getBreakpointId(@NotNull PsiElement breakpointTag) {
             final String url = virtualFile.getPresentableUrl();
             if (virtualFile.isInLocalFileSystem()) { //TODO - we need a better way to match source to target
                 sourceLocations = List.of(
-                    String.format("file:%s", url.replace("src/main/resources", "target/classes")) // file:/absolute/path/to/file.xml
+                    String.format("file:%s", url.replace("src/main/resources", "target/classes")),// file:/absolute/path/to/file.xml
+                    String.format("classpath:%s", url.substring(url.lastIndexOf("src/main/resources/") + 19 ))
```

Comment:
Could you please clarify a bit more by adding a comment

Reply:
According to my observations, Camel and Camel SpringBoot run configurations pull the code from `target/classes`, but Camel Quarkus builds the JAR first and then runs the JAR. So when the debugger calls Camel and requests XML routes dump, the `sourceLocation` attribute value is in form of `classpath:myroutesfile.xml` as opposed to `file:/com/foo/bar/target/classes/myroutesfile.xml`. Since the plugin is generic, i.e. it should support Camel, SpringBoot and Quarkus run configurations, I simply add `classpath:...` to the list of source locations.

Reply:
Would you mind adding this info as comment into the code?

Reply:
done.

Reply:
I found one more related issue, the step over and step into don't work, the problem is here:
https://github.com/camel-tooling/camel-idea-plugin/blob/main/camel-idea-plugin/src/main/java/com/github/cameltooling/idea/runner/debugger/CamelDebuggerSession.java#L862
But I don't know yet how to address it...

Reply:
I planned to work on the plugin next week, if you can wait a bit, I'll look into that too

Reply:
Actually I fixed it too, see my last commit. It is essentially the same problem, classpath vs absolute path.

I feel like the whole debugger code is a bit clunky and could  use some refactoring and refinement, but unfortunately I don't have any time to work on it at least for the next two-three months...